src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -772,28 +700,28 @@ void probe_semi_anti(const RadixContainer<ProbeColumnType>& radix_probe_column,
       } else if constexpr (mode == JoinMode::AntiNullAsFalse) {  // NOLINT - doesn't like else if constexpr
         // no hash table on other side, but we are in AntiNullAsFalse mode which means all tuples from the probing side
         // get emitted.
-        pos_list_local.reserve(partition_end - partition_begin);
-        for (size_t partition_offset = partition_begin; partition_offset < partition_end; ++partition_offset) {
-          auto& probe_column_element = partition[partition_offset];
+        pos_list_local.reserve(elements.size());
+        for (auto partition_offset = size_t{0}; partition_offset < elements.size(); ++partition_offset) {
+          auto& probe_column_element = elements[partition_offset];
           pos_list_local.emplace_back(probe_column_element.row_id);
         }
       } else if constexpr (mode == JoinMode::AntiNullAsTrue) {  // NOLINT - doesn't like else if constexpr
         // no hash table on other side, but we are in AntiNullAsTrue mode which means all tuples from the probing side
         // get emitted. That is, except NULL values, which only get emitted if the build table is empty.
         const auto build_table_is_empty = build_table.row_count() == 0;
-        pos_list_local.reserve(partition_end - partition_begin);
-        for (size_t partition_offset = partition_begin; partition_offset < partition_end; ++partition_offset) {
-          auto& probe_column_element = partition[partition_offset];
+        pos_list_local.reserve(elements.size());
+        for (auto partition_offset = size_t{0}; partition_offset < elements.size(); ++partition_offset) {
+          auto& probe_column_element = elements[partition_offset];
           // A NULL on the probe side never gets emitted, except when the build table is empty.
           // This is because `NULL NOT IN <empty list>` is actually true
-          if ((*probe_column_null_values)[partition_offset] && !build_table_is_empty) {
+          if (null_values[partition_offset] && !build_table_is_empty) {
             continue;
           }
           pos_list_local.emplace_back(probe_column_element.row_id);
         }
       }
 
-      pos_lists[current_partition_id] = std::move(pos_list_local);
+      pos_lists[partition_idx] = std::move(pos_list_local);
```

Comment:
ass?

Reply:
please clarify

Reply:
/idx/index/ ?

Reply:
"idx" is heavily used across the code base (888 matches across 68 files). I don't have a strong feeling about it, but this PR is probably not the right place to start that battle.